packages/twenty-server/src/modules/mortgage/standard-objects/mortgage.workspace-entity.ts

```
@@ -0,0 +1,211 @@
+import { msg } from '@lingui/core/macro';
+import {
+  ActorMetadata,
+  CurrencyMetadata,
+  FieldMetadataType,
+  RelationOnDeleteAction,
+} from 'twenty-shared/types';
```

ESLint

[error] Imports "ActorMetadata" and "CurrencyMetadata" are only used as type.

(@typescript-eslint/consistent-type-imports)